test/acl.test.js

```diff
@@ -6,6 +6,9 @@
 /*global describe, beforeEach, afterEach, it */
 'use strict';
 var path = require('path');
+var SG = require('strong-globalize');
+SG.SetRootDir(path.join(__dirname, '..'), {autonomousMsgLoading: 'all'});
```

Comment:
Please extract these two lines into `lib/globalize.js`, see how loopback does it in https://github.com/strongloop/loopback/blob/31913c048dfc88ba4993a6204d74bb45b5c32950/lib/globalize.js.